run/trino-420/config/tpcds/wp3_rw_concurrency.yaml

```diff
@@ -42,7 +23,9 @@ phases:
   - tasks:
     - template_id: single_user
   - tasks:
-    - template_id: optimize
+    - template_id: optimize_split
```

Comment:
Why the change to a different template? And should it be replaced everywhere?

Reply:
The change is explained in the [README file](/run/trino-420/azure-pipelines/README.md).
Given that this limitation is specific to Trino, I believe there's no need to apply the same approach to Spark immediately. (Maybe it's worth considering in some scenarios to ensure a more accurate comparison between the two though.)